mysite/manage.py

```diff
@@ -2,7 +2,7 @@
 """Django's command-line utility for administrative tasks."""
 import os
 import sys
-from mysite import start_redis_server
+#from mysite import start_redis_server
```

Comment:
should this be added back in ?

Reply:
let it be commmented out so we some one want to use auto startup of redis they can use.